src/routes/routes.ts

```diff
@@ -0,0 +1,8 @@
+import { Router } from "express";
+import { UserController } from "../controllers/UserController";
+
+const routes = Router()
+
+routes.post('/signup', new UserController().signup)

```

Comment:
vc deve criar o middleware para verificar o request e colocar ele aqui. vai ficar algo como:

routes.post('/signup', meumiddleware, new User Controller().signup)

outro ponto, por questão de legibilidade vc deveria iniciar uma instancia do controller fora da chamada mesmo, igual vc ta fazendo ali na linha 4 com o routes, ficaria algo como:

const userController = UserController();

routes.post('/signup', <seumiddleware>, userController.signup)

Reply:
Feito no commit including middleware in the route.